tools/test-builds/browser-agent-wrapper/webpack.config.js

```diff
@@ -0,0 +1,166 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+const isProduction = process.env.NODE_ENV == 'production'
+const htmlTemplate = (script) => `<html>
+  <head>
+    <title>RUM Unit Test</title>
+    {init}
+    {config}
+    <script src="${script}.js"></script>
+    {script-injection}
+  </head>
+  <body>
+    <h1>This is a generic page that is instrumented by the NPM agent</h1>
+  </body>
+</html>`
+const workerHtmlTemplate = `<html>
+  <head>
+    <title>RUM Unit Test</title>
+    {init}
+    {config}
+    {worker-commands}
+    {script-injection}
```

Comment:
`script-injection` should go below the loader, like existing instrumented.html

Reply:
There is no loader in the worker example html. The worker is loaded in the worker script, not the page.

Reply:
Meant below worker script in any case

Reply:
can you check now and see if I got it corrected